.chloggen/featureremovedefaultallowedFlexVolumes.yaml

```diff
@@ -0,0 +1,5 @@
+change_type: 'deprecation'
+component: 'chart'
+note: "Remove by default empty allowedFlexVolumes"
+issues: []
```

Comment:
Looks like an issue # is necessary to pass changelog validation. Can you create an Issue for this PR and link it here?

Reply:
Since this is a simple change, you can just put the PR link here so the validation passes.